WBooks/TabBar/TabBarController.swift

```diff
@@ -13,19 +13,44 @@ class TabBarController: UITabBarController {
         super.viewDidLoad()
         
         let _libraryViewModel = LibraryViewModel()
-        let navLibrary = createNavigationItem(controller: LibraryViewController(libraryViewModel: _libraryViewModel), tag: 0, title: "TITLE_LIBRARY", imageUnselect: .libraryUnSelect, imageSelect: .librarySelect)
+        let libraryViewController = LibraryViewController(libraryViewModel: _libraryViewModel)
+        let navLibrary = createNavigationItem(controller: libraryViewController,
+                                              tag: 0,
+                                              title: "TITLE_LIBRARY",
```

Comment:
Don't we need to localized the title?

Reply:
Of course, the titles are located, only that the string by which it will be located is sent by parameter to the createNavigationItem method

![Screen Shot 2021-08-23 at 1 58 02 PM](https://user-images.githubusercontent.com/88329934/130502499-558d67ec-6c09-44cf-99f6-6396acc0d6ae.png)